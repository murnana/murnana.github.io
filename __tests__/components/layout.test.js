import React from "react"
import renderer from "react-test-renderer"
import Layout from "../../src/components/layout"

describe("Layout", () => {
  it("render", () => {
    const tree = renderer
      .create(
        <Layout>
          <div>
            <h1>Hello World!</h1>
          </div>
        </Layout>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
