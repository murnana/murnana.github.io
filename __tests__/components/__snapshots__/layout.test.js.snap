// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout render 1`] = `
<div
  className="layout"
>
  <button
    className="menu"
    onClick={[Function]}
  >
    Menu
  </button>
  <main>
    <div>
      <h1>
        Hello World!
      </h1>
    </div>
  </main>
</div>
`;
